force-app/main/default/lwc/edaSettings/edaSettings.html

```diff
@@ -2,13 +2,13 @@
     <template if:true={currentUserHasAccess}>
         <div class="slds-grid slds-theme_default eda-height_full">
             <div
-                class="slds-col slds-p-top_x-small slds-no-flex slds-size_4-of-12 slds-scrollable_y eda-height_full"
+                class="slds-col slds-no-flex slds-size_4-of-12 slds-scrollable_y slds-p-top_x-small"
                 data-qa-locator="edaSettingsNavigation"
             >
                 <c-eda-settings-navigation onsettingsnavigation={handleSettingsNavigation}></c-eda-settings-navigation>
             </div>
             <div
-                class="slds-col slds-no-flex slds-size_8-of-12 slds-scrollable_y eda-height_full"
```

Comment:
we dont need eda-height_full here since upper level div is having it.

Reply:
We do because each content area has its own definition of size. So what we're specifying in this instance is that the overall settings container needs to take up 100% of the height of the content area it's in and then within that the nav and page need to take up the height of the settings container. Then they need to be scrollable on the y axis.